src/main/kotlin/no/nav/personbruker/dittnav/eventaggregator/metrics/kafka/KafkaEventCounterService.kt

```diff
@@ -43,8 +43,11 @@ class KafkaEventCounterService(val environment: Environment) {
         return result
     }
 
-    fun countBeskjeder(): Long {
+    fun countBeskjeder(topicToCount: String? = null): Long {
         return try {
+            if(!topicToCount.isNullOrBlank()) {
+                beskjedConsumer = createCountConsumer(EventType.BESKJED, topicToCount)
```

Comment:
Kan ikke dette gi litt uventede resultater? At hvis noen andre har telt på en "annen topic" enn standard, så vil neste kall telle på "annen topic" som forrige kall spesifiserte. Var det meningen?

Reply:
Kanskje det er like greit å lage en egen tellerService-instans hvor man kan opprette en ny klient hver gang? Tenker da i tillegg til denne som går mot de vanlige topic-ene. Det er tungt å opprette kafka-consumer-ene, og da slipper man at det skjer hele tiden.

Reply:
Ja, bra poeng. Hvor tenker du at den tellerservice-en skal instansieres?

Reply:
Jeg tenker at de kan instansieres på samme sted, i appContext-en. KafkaEventCounterService blir forbli slik den var. Den nye service-en kan være helt lik bare at den oppretter en ny createCountConsumer hver gang.
Selve teller-logikken som ligger i KafkaEventCounterService kan jo brukes av begge klassene.

Reply:
Du kan jo strukturere det slik du synes passer best. Det jeg ønsker er bare en rendyrket telle-service for hvert use-case, en for de vanlige topic-ene og en for custom-topics.

Reply:
Skjønner. Det fungerer fint det - å skille de to tingene fra hverandre.